WooCommerce/Classes/ViewRelated/Products/Categories/ProductCategoryListViewController.swift

```diff
@@ -48,6 +67,30 @@ private extension ProductCategoryListViewController {
         tableView.delegate = self
         tableView.removeLastCellSeparator()
     }
+
+    func configureSearchBar() {
+        searchBar.isHidden = !configuration.searchEnabled
+        searchBar.placeholder = Localization.searchBarPlaceholder
+    }
+
+    func configureClearSelectionButton() {
+        clearSelectionButton.setTitle(Localization.clearSelectionButtonTitle, for: .normal)
+        clearSelectionButton.applyLinkButtonStyle()
+        clearSelectionButton.addAction(UIAction { [weak self] _ in
+            self?.viewModel.resetSelectedCategoriesAndReload()
+        }, for: .touchUpInside)
+
+        selectedListSubscription = viewModel.$selectedCategories
+            .map { [weak self] selectedItems -> Bool in
+                guard let self = self, self.configuration.clearSelectionEnabled else {
+                    return true
+                }
+                return selectedItems.isEmpty
+            }
+            .sink { [weak self] shouldHideClearSelection in
```

Comment:
If our intention is to only change the visibility of `clearSelectionButtonBarView`, what do you think about using `assign` instead of `sink`?

IMO, by using `assign` we can,
1. Remove `selectedListSubscription: AnyCancellable?` variable.
2. Clearly state that we are assigning `clearSelectionButtonBarView.isHidden` in this block of code and no other work happening inside.